Swrve-mParticle.podspec

```diff
@@ -1,5 +1,5 @@
 Pod::Spec.new do |s|
-    s.name             = "mParticle-Swrve"
+    s.name             = "Swrve-mParticle"
     s.version          = "1.0"
```

Comment:
~~ version number should be 0.1.0 as discussed

Reply:
done